packages/grafana-ui/src/components/Table/types.ts

```diff
@@ -41,6 +41,17 @@ export interface TableCellProps extends CellProps<any> {
 
 export type CellComponent = FC<TableCellProps>;
 
+/**
+ * Not used yet
+ */
+export interface CustomCellRendererProps {
+  field: Field;
+  value: any;
+  // what else to pass in order to make it more agnostic to table internals?
```

Comment:
field & value maybe be enough. Can use a transformer to pass what's needed as a value